dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/RedshiftClusterNodesTask.java

```diff
@@ -58,4 +58,16 @@ protected Void doRun(TaskRunContext context, @Nonnull ByteSink sink, Handle hand
             .collect(toList()));
     return null;
   }
+
+  private String getRequestDescription() {
```

Comment:
Consider renaming to `toCallDescription` method for consistency with:

https://github.com/google/dwh-migration-tools/blob/d77780f4825a2f3130d4e8942c5a4c1ded8b82c6/dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/bigquery/AbstractBigQueryConnector.java#L174-L179

or:
https://github.com/google/dwh-migration-tools/blob/d77780f4825a2f3130d4e8942c5a4c1ded8b82c6/dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hive/AbstractHiveConnector.java#L242-L247

Reply:
👍